sdktests/server_side_persistence_base.go

```diff
@@ -1,6 +1,17 @@
 package sdktests
 
+// NOTE: You may note that these tests do not follow the same pattern as the
+// other tests in this repository.
+//
+// Historically, tests in this repository have suffered from onion-like
+// nesting. The further you have to dig, the more you cry.
+//
+// In the time honored tradition of every other language on the planet, we are
```

Comment:
![image](https://github.com/user-attachments/assets/79cd118b-2ebb-42a0-95b8-5dd60e047a8f)

Reply:
This was @cwaldren-ld 's prompting. Way to be the change you want to see. 😄